prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctions.kt

```diff
@@ -121,7 +121,7 @@ object CustomFHIRFunctions : FhirPathFunctions {
                     "returns the number of years, months if less than a year, weeks if less" +
                         " than a month, and days if less than a week that a person is old for a date resource",
```

Comment:
Not sure if it'd be too verbose, or if we just want to keep the parameters hidden for now, but we could consider adding to the description to describe the parameters.